include/psa/crypto.h

```diff
@@ -3781,6 +3781,85 @@ psa_status_t psa_key_derivation_output_key(
  *                          the policy must be the same as in the current
  *                          operation.
  * \param[in,out] operation The key derivation operation object to read from.
+ * \param[in] custom        Customization parameters for the key generation.
```

Comment:
`custom_data` is the variable-length part of `custom`. We could be very precise and call the parameters `fixed_size_custom_parameters` and `variable_size_custom_parameters`, but that's too verbose even for me. We [discussed](https://github.com/ARM-software/psa-api/issues/167#issuecomment-2104602992) the names when writing the [spec](https://github.com/ARM-software/psa-api/pull/194). Obviously we can still change them but I'd like a stronger argument to change them.

Reply:
Sorry I forgot that there was a design document when I first reviewed this PR.
I would have preferred to use `params`/`params_data`/`params_data_len` as suggested in the [specs](https://github.com/ARM-software/psa-api/issues/167#issuecomment-2104602992) instead of `custom`/`'custom_data`/`custom_data_len`, but at least the 3 names follow the same pattern, so I'm OK with that in the end.